<?xml version="1.0"?>
<launch>
  <arg name = "module_ids" default=""/>
  <arg name = "real_machine" default="true"/>
  <arg name = "approach_mode" default="nav"/>
  <node pkg="ninja" type="assembly_motion.py" name="" output="screen" >
    <param name = "module_ids" value="$(arg module_ids)"/>
    <param name = "real_machine" value="$(arg real_machine)"/>
    <param name = "approach_mode" value="$(arg approach_mode)"/>
  </node>
</launch>
